wiki/Storyboards/en.md

```diff
@@ -1,17 +1,19 @@
+A **Storyboard** (SB) is a custom-made animated background that accompanies a [Beatmap](/wiki/Beatmaps), often for decorative and sometimes for gameplay purposes. A storyboard can be comprised of just about anything, but usually they consist of visual effects designed to make a map more aesthetically pleasing and distinctive to users. Storyboards are inspired from the animated backgrounds found in [Osu! Tatakae! Ouendan](/wiki/Glossary#ouendan). Like their inspiration, Storyboards are able to adjust to the player's performance in the game and display different things depending on how well the player is doing.
 
-Storyboards are often very difficult to make and require considerable time and expertise in standard animation and in the creation of graphics.
+Storyboards are stored in Beatmap folders as either standalone [`.osb` files](/wiki/osu!_File_Formats/Osb_(file_format)) or extensions to the `[Events]` section of a [`.osu` file](/wiki/osu!_File_Formats/Osu_(file_format)). Because of this, it's possible to create different storyboards for different maps within a [Beatmapset](/wiki/Glossary#beatmapset).
+
+Storyboards are often very difficult to make and require considerable time and expertise in standard animation and in the creation of graphics. osu! offers a [Design Editor](/wiki/Beatmap_Editor/Design) inside of the [Beatmap Editor](/wiki/Beatmap_Editor) to aid the creation of Storyboards, but most avid Storyboard producers opt to instead program via [Storyboard scripting](/wiki/Storyboard_Scripting) directly. Many creators choose to write programs in full-featured programming languages to generate Storyboards scripts, because complex visual effects can require a great amount of Storyboard code to produce.
 
 Full Storyboards
 ----------------
 
-With the advent of the Storyboard Editor to create simple graphical effects in Beatmaps, the full list of [Ranked](/wiki/Beatmaps) Beatmaps with storyboards [is rather long](https://osu.ppy.sh/p/beatmaplist?s=7&o=1&q=&r=0&g=0&la=0&sb=1). Therefore, this page only lists those Beatmaps who have “full” storyboards. If you want more, go to the [full maps list](/wiki/Storyboards/Maps).
+With the advent of the Storyboard Editor to create simple graphical effects in Beatmaps, the full list of Ranked Beatmaps with storyboards [is rather long](https://osu.ppy.sh/p/beatmaplist?s=7&o=1&q=&r=0&g=0&la=0&sb=1). Therefore, this page only lists those Beatmaps who have “full” storyboards. If you want more, go to the [full maps list](/wiki/Storyboards/Maps).
```

Comment:
Storyboard Editor -> Design editor.

Beatmaps -> beatmaps

Ranked Beatmaps -> ranked beatmaps

Beatmaps -> beatmaps

“full” -> "full" (Replace that fancy " from MSWord or something with the simple one from the keyboard).

----

What the heck was I thinking that time. 🍶